src/App.jsx

```diff
@@ -11,36 +11,38 @@ import ForgotPassword from './components/Authentication/ForgotPassword';
 import EmailAction from './components/Authentication/EmailAction';
 import AUTH_ROLES from './utils/auth_config';
 import ProtectedRoute from './utils/ProtectedRoute';
+import Catalog from './pages/Catalog/Catalog';
 
 const { ADMIN_ROLE, USER_ROLE } = AUTH_ROLES.AUTH_ROLES;
 
 const App = () => {
   return (
-      <ChakraProvider>
-        <EmailSending />
-        <CookiesProvider>
-          <Router>
-            <Routes>
-              <Route exact path="/" element={<Login />} />
-              <Route exact path="/logout" element={<Logout />} />
-              <Route exact path="/forgotpassword" element={<ForgotPassword />} />
-              <Route exact path="/register" element={<Register />} />
-              <Route exact path="/emailAction" element={<EmailAction redirectPath="/" />} />
-              <Route
-                exact
-                path="/dashboard"
-                element={
-                  <ProtectedRoute
-                    Component={Dashboard}
-                    redirectPath="/"
-                    roles={[ADMIN_ROLE, USER_ROLE]}
-                  />
-                }
-              />
-            </Routes>
-          </Router>
-        </CookiesProvider>
-      </ChakraProvider>
+    <ChakraProvider>
```

Comment:
This could have been a prettier change on my end to fix the formatting; should I revert this?